R/createCovariates.R

```diff
@@ -66,26 +66,27 @@
 #' covariate and examples.
 #' @keywords datagen IO
 #' @examples
-#' 
-#' 
-#'   ## unit tests for the covariates component of the MSToolkit package 
+#'
+#'
+#'   ## unit tests for the covariates component of the MSToolkit package
 #'   \dontrun{
 #'     file.show( system.file( "Runit", "runit.data.covariates.R", package = "MSToolkit") )
-#'   
-#'     wPath <- system.file( "Runit", "data", "createCovariates", package = "MSToolkit") 
-#'     dAll <- createCovariates( 30, 
+#'
+#'     wPath <- system.file( "Runit", "data", "createCovariates", package = "MSToolkit")
```

Comment:
Should we move the Runit/data/createCovariates file to some other location? Might be clearer to avoid using Runit folders since we've moved to testthat.